### PR TITLE
document currently active roles

### PR DESCRIPTION
**Pull request by ultrasaurus**
add details for 
- triage team (https://github.com/cncf/sig-security/issues/198)
- security assessment facilitator

### CODE REVIEW

**Comment by JustinCappos** [comment]
LGTM

File: assessments/guide/security-reviewer.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-operation. This is meant to provide an independent analysis and estimation
-of the above information.  The goal is to ask questions to understand if
-there are hidden assumptions, underestimated risk, or design issues that
-harm the security of the project.  It may be useful to reach out to
-community members to understand the answers to some questions, especially
-involving deployment scenarios and the impact of attacks.
+# Security Reviewer Role
+
+Reviewers will try to understand the system and probe its security.
+Specifically design level issues are meant to be addressed as well as high
```
Comment (TheMoxieFox) [inline assessments/guide/security-reviewer.md]: Reads strangely. Recommend reword

File: assessments/guide/security-reviewer.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+level problems with the project’s setup and operation. This is meant to 
+provide an independent analysis and estimation of the above information.
```
Comment (TheMoxieFox) [inline assessments/guide/security-reviewer.md]: Recommend defining above information more clearly, as there is much information above this section

File: assessments/guide/security-reviewer.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
-community members to understand the answers to some questions, especially
-involving deployment scenarios and the impact of attacks.
+# Security Reviewer Role
+
+Reviewers will try to understand the system and probe its security.
+Specifically design level issues are meant to be addressed as well as high
+level problems with the project’s setup and operation. This is meant to 
+provide an independent analysis and estimation of the above information. 
+The goal is to ask questions to understand if there are hidden assumptions,
+underestimated risk, or design issues that harm the security of the project.
```
Comment (TheMoxieFox) [inline assessments/guide/security-reviewer.md]: Recommend adding "or risk" after harm

File: governance/roles.md [hunk 4]
```diff
@@ -81,12 +83,61 @@ Project Leads are nominated and approved by the following process:
...
+
+### Security Assessment Facilitator
+
+[Security Assessments](/assessments) are part of the on-going work of the group
+and led by a Security Assessment Facilitator, who will:
+
+* coordinate security review leads for upcoming security assessments.
+* identify and recommend security reviewers.
+* contribute to process improvements.
+* review and merge PRs in the /assessments directory (ensuring co-chair review
```
Comment (TheMoxieFox) [inline governance/roles.md]: I don't see permissions assigned to this role above

Comment (ultrasaurus) [reply governance/roles.md]: *this* is the specification of the permissions -- it's an attempt to scope what this person should do (and describe the required reviews)

Comment (TheMoxieFox) [reply governance/roles.md]: I meant above,. If they are meant to merge in PRs they aren't listed as push or a corresponding role with humans and permission to push.  It currently implies the assessment facilitator can only be one of the previous humans with existing push.

**Pull request merged**